cpp/windows_openssl.mk

```diff
@@ -21,21 +21,21 @@ ONLY_PUBSUB_API = 0
 USE_PROXY = 1
 !endif
 
-!ifndef USE_GRANT_TOKEN
-USE_GRANT_TOKEN = 1
+!ifndef USE_PAM_V3
+USE_PAM_V3 = 1
 !endif
 
 !if $(USE_PROXY)
-PROXY_INTF_SOURCEFILES = ..\core\pubnub_proxy.c ..\core\pubnub_proxy_core.c ..\core\pbhttp_digest.c ..\core\pbntlm_core.c ..\core\pbntlm_packer_sspi.c ..\windows\pubnub_set_proxy_from_system_windows.c 
+PROXY_INTF_SOURCEFILES = ..\core\pubnub_proxy.c ..\core\pubnub_proxy_core.c ..\core\pbhttp_digest.c ..\core\pbntlm_core.c ..\core\pbntlm_packer_sspi.c ..\windows\pubnub_set_proxy_from_system_windows.c
 !endif
 
-!if $(USE_GRANT_TOKEN)
-GRANT_TOKEN_SOURCEFILES = ..\core\pbcc_grant_token_api.c ..\core\pubnub_grant_token_api.c ..\lib\cbor\cborparser.c ..\lib\cbor\cborerrorstrings.c ..\lib\cbor\cborparser_dup_string.c   
+!if $(USE_PAM_V3)
+PAM_V3_SOURCEFILES = ..\core\pbcc_grant_token_api.c ..\core\pubnub_grant_token_api.c ..\lib\cbor\cborparser.c ..\lib\cbor\cborerrorstrings.c ..\lib\cbor\cborparser_dup_string.c ../core/pubnub_revoke_token.c ../core/pbcc_revoke_token.c
```

Comment:
for revoke files, we need to fix the paths. It should have backward slash, not forward slash.

Reply:
These lines has been changed to support fine granular control over enabled PAMv3 features. Paths has been fixed.